xml/schema/Core/Dashboard.xml

```diff
@@ -35,6 +35,7 @@
     <table>civicrm_domain</table>
     <key>id</key>
     <add>3.1</add>
+    <onDelete>CASCADE</onDelete>
```

Comment:
is this right?

Reply:
well the column def is a NOT NULL column so its the only one that seemed to make sense to me

Reply:
ok